x-pack/plugin/transform/src/internalClusterTest/java/org/elasticsearch/xpack/transform/integration/TransformNoTransformNodeIT.java

```diff
@@ -30,7 +31,10 @@
 public class TransformNoTransformNodeIT extends TransformSingleNodeTestCase {
     @Override
     protected Settings nodeSettings() {
-        return Settings.builder().put(NodeRoleSettings.NODE_ROLES_SETTING.getKey(), "master, data, ingest").build();
+        return Settings.builder()
+            .put(NodeRoleSettings.NODE_ROLES_SETTING.getKey(), "master, data, ingest")
+            .put(XPackSettings.SECURITY_ENABLED.getKey(), false) // TODO Change this to run with security enabled
```

Comment:
Since you already did the work to identify the tests that maybe need to work with Security, do you mind opening a meta issue with checkboxes with all of these (and referring the issue in comments)?
I think this would be very helpful in tracking the work :)

Reply:
Good call I added https://github.com/elastic/elasticsearch/issues/75940, will mention it in all relevant places